index.html

```diff
@@ -627,15 +627,18 @@ <h2>Request Headers for an LDPRv</h2>
           <section id="LDPRv-get-response">
             <h2>Response Headers</h2>
             <p>
-              The response to a GET request on an <a>LDPRv</a> MUST include a <code>Link: rel="original timegate"</code>
-              header referencing itself, and at least one <code>Link: rel="timemap"</code> header referencing an
-              associated <a>LDPCv</a>. It is the presence of these headers that indicates that the resource is
-              versioned.
+            The response to a GET request on an <a>LDPRv</a> MUST include the following <code>Link</code> headers:
             </p>
+            <ul>
+              <li>A <code>Link: rel="original timegate"</code> header referencing itself</li>
+              <li>A <code>Link: &lt;http://mementoweb.org/ns#TimeMap&gt;; rel="type"</code> header</li>
```

Comment:
This should be `TimeGate` not `TimeMap`, right?

Reply:
agreed, well spotted!